### DISCUSSION TITLE
Need help to setup Expo and  Rn Mapbox

### DISCUSSION THREAD

**Discussion by TheUncharted** [rnmapbox/maps#3086] [Q&A] [answered]
Hi 

I created a small project and i had some trouble to install it at first and solved it with a webpack config and some dependencies like worker-loader.
Now I'm able to display a map but when I try to display an PointAnnotation

```typescript
const App = () => {
  const pointAnnotation = useRef<Mapbox.PointAnnotation>(null);
  return (
    <Mapbox.MapView style={styles.map}>
      <Mapbox.Camera
        defaultSettings={{
          centerCoordinate: [-74.00597, 40.71427],
          zoomLevel: 12,
        }}
      />
      <Mapbox.PointAnnotation
        id="annotation"
        coordinate={[-74.00597, 40.71427]}
        title="test"
        draggable
        ref={pointAnnotation}
      >
        <Mapbox.Callout title="This is a sample loading a remote image" />
      </Mapbox.PointAnnotation>
    </Mapbox.MapView>
  );
};
```

I get those errors

![image](https://github.com/rnmapbox/maps/assets/24917492/22b8895f-e483-4473-80a6-4dc490ab041e)

Any idea on how to solve it ?

To reproduce it : [Repo](https://github.com/TheUncharted/testmapbox)

**Comment by mfazekas** [comment, marked as answer]
@TheUncharted are you trying to run it on the web? Web support is very incomplete and experimental

Pls add the command to build/start.

**Reply by TheUncharted** [reply]
@mfazekas 
yes i try to run it on the web with : yarn web
Is there is a solution if I want to develop on web before mobile ?
